infra-templates/container-crontab/1/rancher-compose.yml

```diff
@@ -1,6 +1,6 @@
 .catalog:
   name: "Rancher Container Crontab"
-  version: "v0.1.0-rancher1"
+  version: "v0.2.0-rancher1"
   description: "Service that will start, stop or restart containers"
   minimum_rancher_version: v1.4.0
```

Comment:
does this work in all versions or only certain versions?

also, can you add `maximum_rancher_version` to the template in folder `0`